### PR TITLE
Unify the usage of arrow functions

### PR DESCRIPTION
**Pull request by furkanpoyraz**
Replaced es5 functions with arrow functions.
+1 for consistency. :)

### CODE REVIEW

**Comment by timdorr** [comment]
Thanks!

File: examples/universal/common/containers/App.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const mapStateToProps = (state) => ({
   counter: state.counter
 })
 
-function mapDispatchToProps(dispatch) {
+const mapDispatchToProps = () => {
```
Comment (mindjuice) [inline examples/universal/common/containers/App.js]: dispatch?

Comment (markerikson) [inline examples/universal/common/containers/App.js]: Just fixed it on master - thanks for noticing that!

Comment (furkanpoyraz) [inline examples/universal/common/containers/App.js]: Oops! Thanks for the notice.

**Pull request merged**